recipes-bsp/firmware-imx/firmware-imx_8.23.bb

```diff
@@ -16,8 +16,9 @@ do_install() {
     # SDMA Firmware section
     install -d ${D}${nonarch_base_libdir}/firmware/imx/sdma
     install -m 0644 ${S}/firmware/sdma/* ${D}${nonarch_base_libdir}/firmware/imx/sdma
-    rm -f ${D}${nonarch_base_libdir}/firmware/imx/sdma/sdma-imx6q.bin
-    rm -f ${D}${nonarch_base_libdir}/firmware/imx/sdma/sdma-imx7d.bin
+    # Comment these lines to use sdma-imx6q/7d.bin from here and not linux-firmware
+    #rm -f ${D}${nonarch_base_libdir}/firmware/imx/sdma/sdma-imx6q.bin
+    #rm -f ${D}${nonarch_base_libdir}/firmware/imx/sdma/sdma-imx7d.bin
```

Comment:
Please let our change in this hook.

Reply:
If `rm` is used for sdma FWs, they are never be able to use from official firmware-imx recipe, why is it good?

Reply:
We use the `linux-firmware` provided ones.